### PR TITLE
windowsdesktop-runtime: Update to version 5.0.3

### PR DESCRIPTION
**Pull request by x-magic**
Finally got it fixed...

### CODE REVIEW

**Comment by x-magic** [comment]
This replaces #5637, which matches the latest desktop runtime version. 
LTS version (3.x) is added in https://github.com/ScoopInstaller/Versions/pull/260

**Comment by Ash258** [comment]
/verify checkver has to fail unless there is cdn for latest endpoint

image

**Comment by x-magic** [comment]
> /verify checkver has to fail unless there is cdn for latest endpoint
> 
> image

Even the uncached feed is still 5.0.2. 
https://dotnetcli.blob.core.windows.net/dotnet/WindowsDesktop/Current/latest.version
image

Can't do anything but wait

**Comment by x-magic** [comment]
Changed Checkver URL to non-cached one from https://github.com/dotnet/install-scripts/blob/master/src/dotnet-install.ps1
As suggested from last comment, unless MS update their feed, check will fail, can't do anything about unless remove checkver.

File: bucket/windowsdesktop-runtime.json [hunk 1]
```diff
@@ -1,39 +1,54 @@
...
+            "url": "https://dotnetcli.blob.core.windows.net/dotnet/WindowsDesktop/5.0.4/windowsdesktop-runtime-5.0.4-win-x64.exe",
+            "hash": "sha512:163c6b9810e8aa66dca7caf20e4052df41d8ca844932ae1fcdcee3207f242f9fa455a976720c7e1e8473a19cf413d593ae46fe46fddee5ed458d2c97695dbb9d"
         },
         "32bit": {
-            "url": "https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.10/windowsdesktop-runtime-3.1.10-win-x86.exe",
-            "hash": "sha512:1697e69644a1df46922e440c1ea16e583660f79f3b9b109b493a24835ae543ea6180c5167d6dbe89d8dd8991f865dba717525e333e4a2f32319733afa6a320b5"
+            "url": "https://dotnetcli.blob.core.windows.net/dotnet/WindowsDesktop/5.0.4/windowsdesktop-runtime-5.0.4-win-x86.exe",
+            "hash": "sha512:b0bd45e1d234ad74e770a7e1acea527f9e84af39da2cdecc6b4ab5c3f855b7f90d5ce7114f1500a9f9163de125cb87a6e4eb99a35d03ba5369db163cce8e3a87"
+        },
+        "arm64": {
```
Comment (Ash258) [inline bucket/windowsdesktop-runtime.json:19]: ????

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by x-magic** [comment]
Updated to version 5.0.4.

File: bucket/windowsdesktop-runtime.json (hunk 1 above)
Comment (x-magic) [reply bucket/windowsdesktop-runtime.json:19]: ?

Comment (x-magic) [reply bucket/windowsdesktop-runtime.json:19]: So the checking script won't recognise arm64? Fine...

**Comment by x-magic** [comment]
> /unsubscribe /block

Okay I give up. This is too hard.

**Pull request closed without merging**